kyaml/runfn/runfn.go

```diff
@@ -518,3 +530,20 @@ func (r *RunFns) ffp(spec runtimeutil.FunctionSpec, api *yaml.RNode, currentUser
 
 	return nil, nil
 }
+
+func getWorkingDirectory(wd string) (string, error) {
+	if wd == "" {
+		return os.Getwd()
```

Comment:
In the spirit of being as restrictive and predictable as possible with exec plugins, could `kustomize fn run` set the working directory as well, and then return an error here instead of having a fallback?

Reply:
Done